packages/labs/observers/src/mutation_controller.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import {ReactiveControllerHost} from '@lit/reactive-element/reactive-controller.js';
+
+export type MutationValueCallback = (
+  ...args: Parameters<MutationCallback>
+) => unknown;
+
+export interface MutationControllerConfig {
+  config: MutationObserverInit;
+  target?: Element;
+  callback?: MutationValueCallback;
+  skipInitial?: boolean;
+}
+
+export class MutationController {
```

Comment:
implements ReactiveController? (not sure how much that helps anything... maybe requires `override`s?)

Reply:
Done.